firmware/user/custom_commands.c

```diff
@@ -52,101 +53,62 @@ typedef struct
+	{32,1,0,6,DUP,3,    5,15,3,44,0,   2,2,96,  4,4,69,  45,1,100,255, 103,DEFAULT_NUM_LEDS/4,     3, 1,0,   0, 0,   0,     1,1, 12, DEFAULT_NUM_LEDS, 0, 0},
+	{32,1,0,6,DUP,3,    5,15,3,44,0,   2,2,96,  4,4,69,  45,1,100,255, 103,DEFAULT_NUM_LEDS/2,     0, 1,0,  65,65,   0,     1,1, 13, DEFAULT_NUM_LEDS, 0, 0},
+	{32,1,0,6,DUP,3,    5,15,3,44,0,   2,2,96,  4,6,69,  45,1,100,255, 103,DEFAULT_NUM_LEDS,       0, 1,1,   0, 0,   0,     1,1,  14,DEFAULT_NUM_LEDS, 0, 0},
+	{32,1,0,6,DUP,3,    5,15,3,44,0,   2,2,96,  4,4,69,  45,1,100,255, 101,DEFAULT_NUM_LEDS/8,     0, 1,0,   1, 31,  1,     1,0,  15,DEFAULT_NUM_LEDS, 0, 0},
+	};
+
+
+uint8_t* gConfigurables[CONFIGURABLES] = { &CCS.gINITIAL_AMP, &CCS.gRMUXSHIFT,  &CCS.gROOT_NOTE_OFFSET, &CCS.gDFTIIR, &CCS.gDFT_UPDATE, &CCS.gFUZZ_IIR_BITS,
+                                           &CCS.gFILTER_BLUR_PASSES, &CCS.gLOWER_CUTOFF, &CCS.gSEMIBITSPERBIN, &CCS.gMAX_JUMP_DISTANCE, &CCS.gMAX_COMBINE_DISTANCE,
+                                           &CCS.gAMP1_ATTACK_BITS, &CCS.gAMP1_DECAY_BITS, &CCS.gAMP_1_MULT, &CCS.gAMP2_ATTACK_BITS, &CCS.gAMP2_DECAY_BITS, &CCS.gAMP_2_MULT, &CCS.gMIN_AMP_FOR_NOTE,
+                                           &CCS.gMINIMUM_AMP_FOR_NOTE_TO_DISAPPEAR, &CCS.gNOTE_FINAL_AMP, &CCS.gNOTE_FINAL_SATURATION,
+                                           &CCS.gNERF_NOTE_PORP, &CCS.gUSE_NUM_LIN_LEDS, &CCS.gSYMMETRY_REPEAT, &CCS.gCOLORCHORD_ACTIVE, &CCS.gCOLORCHORD_OUTPUT_DRIVER, &CCS.gCOLORCHORD_SHIFT_INTERVAL,
+                                           &CCS.gCOLORCHORD_FLIP_ON_PEAK, &CCS.gCOLORCHORD_SHIFT_DISTANCE, &CCS.gCOLORCHORD_SORT_NOTES, &CCS.gCOLORCHORD_LIN_WRAPAROUND, &CCS.gCONFIG_NUMBER,
+                                           &CCS.gNUM_LIN_LEDS, &CCS.gEQUALIZER_SET,  0
+                                         };
+
+char* gConfigurableNames[CONFIGURABLES] = { "gINITIAL_AMP", "gRMUXSHIFT", "gROOT_NOTE_OFFSET", "gDFTIIR", "gDFT_UPDATE", "gFUZZ_IIR_BITS",
+                                            "gFILTER_BLUR_PASSES", "gLOWER_CUTOFF", "gSEMIBITSPERBIN", "gMAX_JUMP_DISTANCE", "gMAX_COMBINE_DISTANCE", "gAMP1_ATTACK_BITS", "gAMP1_DECAY_BITS",
+                                            "gAMP_1_MULT", "gAMP2_ATTACK_BITS", "gAMP2_DECAY_BITS", "gAMP_2_MULT", "gMIN_AMP_FOR_NOTE", "gMINIMUM_AMP_FOR_NOTE_TO_DISAPPEAR", "gNOTE_FINAL_AMP",
+                                            "gNOTE_FINAL_SATURATION", "gNERF_NOTE_PORP", "gUSE_NUM_LIN_LEDS", "gSYMMETRY_REPEAT", "gCOLORCHORD_ACTIVE", "gCOLORCHORD_OUTPUT_DRIVER",
+                                            "gCOLORCHORD_SHIFT_INTERVAL", "gCOLORCHORD_FLIP_ON_PEAK", "gCOLORCHORD_SHIFT_DISTANCE", "gCOLORCHORD_SORT_NOTES", "gCOLORCHORD_LIN_WRAPAROUND",
+                                            "gCONFIG_NUMBER", "gNUM_LIN_LEDS",  "gEQUALIZER_SET", 0
+                                          };
+
 struct CCSettings CCS = {0};
 
-configurable_t gConfigs[CONFIGURABLES] =
```

Comment:
@bbkiwi I'm sad you undid this. It was much easier to keep everything in sync with the structs. Having three arrays that must be in the same order are harder to manage.